src/directives/show.js

```diff
@@ -1,6 +1,11 @@
 import { transitionIn, transitionOut } from '../utils'
 
 export function handleShowDirective(component, el, value, modifiers, initialUpdate = false) {
+    // Resolve any previous pending transitions before starting a new one
```

Comment:
is it relevant only for show? transitions are applied to x-if as well.

Reply:
As i said in #542 we can run `el.transition_remaning()` any time if we need to clear transitions.